docs/model_developers/modules.rst

```diff
@@ -329,3 +329,41 @@ for supplying the agents with the appropriate ``inmanta_plugins`` packages.
 The only exception to this rule is when using the ``inmanta export`` command. It exports a project and all its modules'
 ``inmanta_plugins`` packages to the orchestrator server. When this method is used, the orchestrator does not install any modules
 from the Python package repository but instead contains all Python code as present in the local Python environment.
+
+Configure a project to install modules from a private python package repository
+--------------------------------------------------------------------------------
+
```

Comment:
I think this section could use 2 sub-sections: 
1. The one you already described with `use_config_file` to True
2. Another one with the pip -> index_urls ([ticket](https://github.com/inmanta/inmanta-core/issues/5993) adding this for reference)
The option is used like this in the `project.yml`:
```yml
pip:
    index_urls:
        - <url of the private repository>
```
I don't know how this second section sits with the .netrc file though (i.e. maybe this is a common part to both sub-sections)

Reply:
what happens when `use_config_file` is true and a `index_urls` is provided?

Reply:
Both are used: see [here](https://github.com/inmanta/inmanta-core/blob/c79cc1e1376a13367fa2a3c1949cd4b864ea7fdd/src/inmanta/env.py#L360) and the indexes specified by index_urls will be passed to pip as `--extra-index-url`'s